<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIN解析程序</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 255); border-radius: 10px; padding: 10px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="margin">
     <number>15</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="styleSheet">
       <string notr="true">border-radius: 5px; border: 1px solid #ccc;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_radios">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioSeparate">
        <property name="text">
         <string>Separate</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioMerge">
        <property name="text">
         <string>Merge</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_buttons">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnSelect">
        <property name="text">
         <string>Select</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 5px; padding: 8px 16px; background-color: #4CAF50; color: white;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnParse">
        <property name="text">
         <string>Parse</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 5px; padding: 8px 16px; background-color: #2196F3; color: white;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
